### PR TITLE
ticdc: update ticdc docs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15120
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-faq.md [hunk 1]
```diff
@@ -281,7 +278,11 @@ For TiCDC versions earlier than v6.5.2, it is recommended that you deploy TiCDC
 
 ## What is the order of executing DML and DDL statements?
 
-The execution order is: DML -> DDL -> DML. To ensure that the table schema is correct when DML events are executed downstream during data replication, it is necessary to coordinate the execution order of DDL and DML statements. Currently, TiCDC adopts a simple approach: it replicates all DML statements before the DDL ts to downstream first, and then replicates DDL statements.
+Currently, TiCDC adopts the following order:
+
+1. TiCDC blocks the replication progress of the tables affected by DDL statements until the DDL `CommiTs`. This ensures that DML statements executed before DDL `CommiTs` can be successfully replicated to the downstream first.
+2. TiCDC continue with the replication of DDL statements. If there are multiple DDL statements, TiCDC replicates them in a serial manner.
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:284]:
```suggestion
2. TiCDC continues with the replication of DDL statements. If there are multiple DDL statements, TiCDC replicates them in a serial manner.
```

File: ticdc/ticdc-faq.md [hunk 2]
```diff
@@ -281,15 +278,24 @@ For TiCDC versions earlier than v6.5.2, it is recommended that you deploy TiCDC
...
+3. After the DDL statements are executed in the downstream, TiCDC will continue with the replication of DML statements executed after DDL `CommiTs`.
 
 ## How should I check whether the upstream and downstream data is consistent?
 
 If the downstream is a TiDB cluster or MySQL instance, it is recommended that you compare the data using [sync-diff-inspector](/sync-diff-inspector/sync-diff-inspector-overview.md).
 
 ## Replication of a single table can only be run on a single TiCDC node. Will it be possible to use multiple TiCDC nodes to replicate data of multiple tables?
 
-This feature is currently not supported, which might be supported in a future release. By then, TiCDC might replicate data change logs by TiKV Region, which means scalable processing capability.
+Starting from v7.1.0, TiCDC supports the MQ sink to replicate data change logs at the granularity of TiKV Regions, which achieves scalable processing capability and allows TiCDC to replicate a single table with a large number of Regions. To enable this feature, you can configure the following parameters in the [TiCDC configuration file](/ticdc/ticdc-changefeed-config.md):
```
Comment (qiancai) [inline ticdc/ticdc-faq.md:293]:
```suggestion
Starting from v7.1.0, TiCDC supports the MQ sink to replicate data change logs at the granularity of TiKV Regions, which achieves scalable processing capability and allows TiCDC to replicate a single table with a large number of Regions. To enable this feature, you can configure the following parameter in the [TiCDC configuration file](/ticdc/ticdc-changefeed-config.md):
```

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -46,15 +41,17 @@ A replication task might be interrupted in the following known scenarios:
...
 
 - Replication cannot continue because of incompatible SQL statement(s) in the downstream.
 
     - In this scenario, TiCDC saves the task information. Because TiCDC has set the service GC safepoint in PD, the data after the task checkpoint is not cleaned by TiKV GC within the valid period of `gc-ttl`.
     - Handling procedures:
         1. Query the status information of the replication task using the `cdc cli changefeed query` command and record the value of `checkpoint-ts`.
         2. Use the new task configuration file and add the `ignore-txn-start-ts` parameter to skip the transaction corresponding to the specified `start-ts`.
-        3. Stop the old replication task via HTTP API. Execute `cdc cli changefeed create` to create a new task and specify the new task configuration file. Specify `checkpoint-ts` recorded in step 1 as the `start-ts` and start a new task to resume the replication.
+        3. Pause the replication task by executing `cdc cli changefeed pause -c <changefeed-id>`.
+        4. Specify the new task configuration file with `cdc cli changefeed update -c <changefeed-id> --config <config-file-path>`.
```
Comment (qiancai) [inline ticdc/troubleshoot-ticdc.md:53]:
```suggestion
        4. Specify the new task configuration file by executing `cdc cli changefeed update -c <changefeed-id> --config <config-file-path>`.
```

**Review by sdojjy — APPROVED** [review]

**Comment by qiancai** [comment]
/review check for grammar or spelling issues

File: ticdc/deploy-ticdc.md [hunk 4]
```diff
@@ -16,9 +16,9 @@ In production environments, the recommendations of software and hardware for TiC
 | Red Hat Enterprise Linux | 7.3 or later versions   |
 | CentOS                   | 7.3 or later versions   |
 
-| CPU | Memory | Disk type | Network | Number of TiCDC cluster instances (minimum requirements for production environment) |
+| CPU | Memory | Disk | Network | Number of TiCDC cluster instances (minimum requirements for production environment) |
 | :--- | :--- | :--- | :--- | :--- |
-| 16 core+ | 64 GB+ | SSD | 10 Gigabit network card (2 preferred） | 2 |
+| 16 core+ | 64 GB+ | 500 GB+ SSD disk | 10 Gigabit network card (2 preferred） | 2 |
```
Comment (ran-huang) [inline ticdc/deploy-ticdc.md:21]: SSD = solid state drive，感觉不用再重复 disk 了
```suggestion
| 16 core+ | 64 GB+ | 500 GB+ SSD | 10 Gigabit network card (2 preferred） | 2 |
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**